src/scaladoc/scala/tools/ant/Scaladoc.scala

```diff
@@ -666,7 +666,7 @@ class Scaladoc extends ScalaMatchingTask {
   /** Performs the compilation. */
   override def execute() = {
     val (docSettings, sourceFiles) = initialize
-    val reporter = new ConsoleReporter(docSettings)
+    val reporter = new ScalaDocReporter(docSettings)
```

Comment:
`ant docs` runs this file - so we should use `ScalaDocReporter` here instead to get the new behaviour.